docs/csharp/tutorials/inheritance.md

```diff
@@ -133,12 +133,12 @@ Because of implicit inheritance, you can call any inherited member from a `Simpl
 
 The following table lists the categories of types that you can create in C# and the types from which they implicitly inherit. Each base type makes a different set of members available through inheritance to implicitly derived types.
 
-| Type category | Implicitly inherits from                                                      |
-| ------------- | ----------------------------------------------------------------------------- |
-| class         | <xref:System.Object>                                                          |
-| struct        | <xref:System.ValueType>, <xref:System.Object>                                 |
-| enum          | <xref:System.Enum>, <xref:System.ValueType>, <xref:System.Object>             |
-| delegate      | <xref:System.MulticastDelegate>, <xref:System.Delegate>, <xref:System.Object> |
+| Type category                    | Implicitly inherits from                                                      |
+| -------------------------------- | ----------------------------------------------------------------------------- |
+| class                            | <xref:System.Object>                                                          |
+| struct (excluding `ref` structs) | <xref:System.ValueType>, <xref:System.Object>                                 |
```

Comment:
Needs confirmation that my change is correct before merging.

Reply:
ping @jaredpar @MadsTorgersen 

Is this how we'd want to explain the implicit inheritance for `ref struct` types?

Reply:
@BillWagner Thanks. But why the compiler says the opposite?

![image](https://user-images.githubusercontent.com/31348972/105061743-86dfe300-5a82-11eb-96d6-c34989b1b52d.png)

Reply:
Diagnostic clarity issue. I'm a bit surprised we didn't make that an error though.

Reply:
@jaredpar Thanks. Opened https://github.com/dotnet/roslyn/issues/50609 to improve this scenario.

Reply:
@Youssef1313 Interestingly, if you use the pattern matching `is`, it generates CS8121 (an error):

```csharp
(if s is object o)
```